client/my-sites/domains/domain-management/settings/cards/contact-information/contacts-card.tsx

```diff
@@ -218,6 +219,13 @@ const ContactsPrivacyCard = ( props: ContactsCardProps ) => {
 							) }
 						</p>
 					) }
+					{ pendingContactUpdate && (
+						<p className="contact-information__pending-update-warn">
+							{ translate(
+								"This domain has a pending contact information update. You will be able to update your contact information once the update is complete. If you don't confirm the update, the pending request will be canceled after 5 days."
```

Comment:
```suggestion
								"This domain has a pending contact information update. You will be able to update your contact information once the pending update is complete. If you don't confirm the update, the pending request will be canceled after 5 days."
```
I think adding this makes the sentence a little clearer.

I wonder if we should also add something like "you should have received an email to confirm the contact information update" so users know where they can confirm that.